### DISCUSSION TITLE
Supporting .NET Framework

### DISCUSSION THREAD

**Discussion by craigomatic** [microsoft/semantic-kernel#252] [Polls]
Some prior investigation was done on netstandard 2.0 support over here: https://github.com/microsoft/semantic-kernel/pull/38 - lots of changes in the repository since then, so may be a larger task than the comments there indicated at the time.

Supporting netstandard 2.0 would add support for: .NET Framework 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8, 4.8.1

**Comment by lilhoser** [comment]
Just to follow up here: I ported my app to .NET 7. It took me about a day to work through the problems, and I was fortunate that all of my dependencies have been ported away from .NET framework to .NET.

**Comment by dluc** [comment]
Quick update, SK now supports .NET Standard 2.0. All nugets are 2.0 compatible. In future we will probably use multi-target builds to leverage .NET 8+ features. In the meantime, all new code is and will 2.0 comp.
